lib/foreman_rh_cloud/engine.rb

```diff
@@ -66,7 +66,7 @@ class Engine < ::Rails::Engine
         # Adding a sub menu after hosts menu
         divider :top_menu, caption: N_('RH Cloud'), parent: :configure_menu
         menu :top_menu, :inventory_upload, caption: N_('Inventory Upload'), url: '/foreman_rh_cloud/inventory_upload', url_hash: { controller: :react, action: :index }, parent: :configure_menu
-        menu :top_menu, :insights_hits, caption: N_('Insights'), url: '/foreman_rh_cloud/insights_cloud', url_hash: { controller: :react, action: :index }, parent: :configure_menu
+        menu :top_menu, :insights_hits, caption: N_('Insights'), url: '/foreman_rh_cloud/insights_cloud?isSelectAll=true', url_hash: { controller: :react, action: :index }, parent: :configure_menu
```

Comment:
@ShimShtein do we want it to open the plugin page always with `selectAll=true` from the menu?
or just to apply `selectAll` from the `url` from specific locations such as `host view`?

Reply:
I suppose by default it should be `select_all = false`.